tests/test_regression.py

```diff
@@ -4,40 +4,48 @@
 from pathlib import Path
 from typing import Any
 
+import pytest
+
 from autocorpus.configs.default_config import DefaultConfig
 
 
-def test_autocorpus(data_path: Path):
-    """A regression test for the main Auto-CORPus class on the Auto-CORPus paper.
+@pytest.mark.parametrize(
+    "input_file, config",
+    [
+        ("PMC/Pre-Oct-2024/PMC8885717.html", DefaultConfig.LEGACY_PMC.load_config()),
+        ("PMC/Current/PMC8885717.html", DefaultConfig.PMC.load_config()),
```

Comment:
I'd rather these not have the `.html` on them and then use pathlib's [`with_suffix`](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.with_suffix) (example on below)
```suggestion
        ("PMC/Pre-Oct-2024/PMC8885717", DefaultConfig.LEGACY_PMC.load_config()),
        ("PMC/Current/PMC8885717", DefaultConfig.PMC.load_config()),
```